project-base/storefront/components/Blocks/OrderAction/OrderAction.tsx

```diff
@@ -56,7 +57,11 @@ export const OrderAction: FC<OrderActionProps> = ({
                     className="font-bold uppercase text-dark no-underline"
                 >
                     <>
-                        <Icon iconType="icon" icon="Arrow" className="relative top-0 mr-1 rotate-90 text-greyLight" />
+                        <Icon
+                            iconType="icon"
+                            icon={<Arrow />}
```

Comment:
Could we do it so that when a component is provided as an `icon`, we do not have to provide the `iconType` prop? It could save us some code.

Reply:
Good idea, I've improved it a little bit.